lib/gitsh/tab_completion/dsl/parser.rb

```diff
@@ -72,6 +78,8 @@ def self.parse(tokens, opts = {})
             tokens,
             opts.merge(env: Environment.new(opts.fetch(:gitsh_env))),
           )
+        rescue RLTK::NotInLanguage => error
```

Comment:
Do we want to handle [any of the other three exceptions that RLTK can raise](https://github.com/chriswailes/RLTK/blob/ffb471f52b0a1e12f05927e7e78920727ba4620a/README.md#parsing-exceptions)?

(Maybe not `InternalParserError`.)

Reply:
I don't think we need to:

- `RLTK::BadToken` should only happen if our lexer and parser disagree on what the set of tokens is, and I'd hope that would be caught by an integration test rather than raising at runtime.
- `RLTK::HandledError` should only happen if the parser uses [error productions][1], which currently we don't, not least because the RLTK README warns that they're not well tested, and because I don't fully understand what they're for.

[1]: https://github.com/chriswailes/RLTK/blob/ffb471f52b0a1e12f05927e7e78920727ba4620a/README.md#error-productions

Reply:
Excellent reasons. LGTM!